conjurapi/resource_test.go

```diff
@@ -56,18 +56,27 @@ func conjurSetup() (*Client, error) {
 }
 
 func TestClient_CheckPermission(t *testing.T) {
-	checkAllowed := func(conjur *Client, id string) func(t *testing.T) {
+	checkAllowed := func(conjur *Client, id string, role string) func(t *testing.T) {
 		return func(t *testing.T) {
-			allowed, err := conjur.CheckPermission(id, "execute")
+			allowed, err := conjur.CheckPermission(id, role, "read")
 
 			assert.NoError(t, err)
 			assert.True(t, allowed)
 		}
 	}
 
-	checkNonExisting := func(conjur *Client, id string) func(t *testing.T) {
+	checkNonExisting := func(conjur *Client, id string, role string) func(t *testing.T) {
```

Comment:
It looks like we could consolidate `checkNonExisting` and `checkNoPermission` - they have the same assertions.

Reply:
got it, I condensed these down to "checkNotAllowed"